tests/reg_tests/test_DVConstraints.py

```diff
@@ -93,12 +100,22 @@ def generic_test_base(DVGeo, DVCon, handler, checkDerivs=True, fdstep=1e-4):
 @parameterized_class(
     [
         {
+            # Standard one-level FFD
             "name": "standard",
             "child": False,
+            "multi": False,
         },
         {
+            # Deforming child FFD with a stationary parent FFD
             "name": "child",
             "child": True,
+            "multi": False,
+        },
+        {
+            # One deforming component FFD and a non-intersecting stationary component FFD
```

Comment:
That's correct. Let me know if the updated description is clearer.

Reply:
I see, the updated description is clearer, but why is a not-intersected case chosen as the test case here?

Reply:
The idea behind parameterizing the DVConstraints tests is to use the same reference values for all the tests. This checks that the function values and gradients for a single component of DVGeometryMulti are the same as one instance of DVGeometry.

Also, `applyIC` is False for all constraint point sets. This is because we want constraints like thickness and volume to follow the component geometry, not the surface mesh. So even if the components intersected, there would be no intersection treatment applied to the point sets.

Reply:
I see. Thanks for the clarification!